app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="bg-white p-4 md:p-6 rounded-lg shadow-md md:w-[100%] mt-3">
+  <h1 class="text-3xl md:text-4xl font-semibold mb-3 md:mb-5">Welcome To Recipe Page</h1>
+
+  <div class="mb-3 md:mb-6">
+    <%= link_to new_recipe_path,
+      class: "px-3 md:px-4 py-2 bg-blue-500 hover:bg-blue-600 text-white font-medium rounded-lg cursor-pointer" do %>
+      Add
+      <% end %>
+  </div>
+
+  <div class="recipe-container mt-3 md:mt-5">
+    <% @recipes.each do |recipe| %>
+      <div
+        class="recipe bg-gray-100 p-3 md:p-4 rounded-lg mb-3 md:mb-4 flex flex-col md:flex-row justify-between items-center">
+        <%= link_to recipe_path(recipe), class: "info" do %>
+          <h2 class="text-xl md:text-2xl font-medium mb-2 md:mb-0">
+            <%= recipe.name %>
+          </h2>
+          <p class="text-gray-600 text-sm md:text-lg">Description: <span class="text-xs md:text-sm">
+              <%= recipe.description %>
+            </span>
+          </p>
+          <% end %>
+          <% if can? :destroy, recipe %>
+            <%= form_with model: recipe, url: recipe_path(recipe), method: :delete, class: "remove mt-3 md:mt-0" do |form| %>
+              <%= form.submit "Remove" ,
+                class: "px-3 md:px-4 py-1 bg-red-500 hover:bg-red-600 text-white font-medium rounded-lg cursor-pointer"
+                %>
+                <% end %>
+                <% end %>
+      </div>
```

Comment:
Great work so far 🥇 , one very smaaaaalll change was requested by the previous reviewer and wasn't addressed, please stick to the given wireframe as the positioning of elements in the recipe card is not the same  
![Screenshot from 2024-02-10 22-09-54](https://github.com/BiancaMe/recipe-app/assets/59707859/d9c1dfae-e9c2-49e8-919d-2eb4f01a698c)
![Screenshot from 2024-02-10 22-07-56](https://github.com/BiancaMe/recipe-app/assets/59707859/e7dda6e2-6801-4ab4-b8dc-b693c326a0bf)

Reply:
- Well resolved 👍🏽 
<img width="1432" alt="Screenshot 2024-02-10 at 23 29 30" src="https://github.com/BiancaMe/recipe-app/assets/58692590/4d0782d6-f5d2-4b7b-a3c4-f632b982005f">